scripts/goimport-file.sh

```diff
@@ -19,7 +19,11 @@ case "$OSTYPE" in
         sed -i -e "$sed_expression" $1
         ;;
     "darwin"*)
-        sed -i '' -e "$sed_expression" $1
+        if command -v gsed; then
+            gsed -i -e "$sed_expression" $1
+        else
+            sed -i '' -e "$sed_expression" $1
+        fi
```

Comment:
It's not so much that we should use gsed if it's present, but that `sed` could be either GNU or posix sed and we should detect that.

Reply:
I've updated to check the `sed` version output to see whether it contains *GNU*. While I'm not sure whether the output of the POSIX `sed --version` also output text that will match *GNU*..